aiida_common_workflows/workflows/relax_and_bands.py

```diff
@@ -0,0 +1,324 @@
+            help='The treshold for determining edge cases. Meaning is different depending on bravais lattice.',
+        )
+        spec.input(
+            'seekpath_parameters.with_time_reversal',
+            valid_type=orm.Bool,
+            default=lambda: orm.Bool(True),
+            serializer=to_aiida_type,
+            help='If False, and the group has no inversion symmetry, additional lines are returned.',
+        )
+
+        namspac = spec.inputs.create_port_namespace('relax_inputs')
+        namspac.absorb(CommonRelaxInputGenerator.spec().inputs)
+        namspac['protocol'].non_db = True
+        namspac['spin_type'].non_db = True
+        namspac['relax_type'].non_db = True
+        namspac['electronic_type'].non_db = True
+        namspac['magnetization_per_site'].non_db = True
+        namspac['threshold_forces'].non_db = True
+        namspac['threshold_stress'].non_db = True
+        namspac['engines']['relax']['options'].non_db = True
+
+        namspac2 = spec.inputs.create_port_namespace('bands_inputs')
+        namspac2.absorb(CommonBandsInputGenerator.spec().inputs, exclude=('parent_folder'))
+        namspac2['engines']['bands']['options'].non_db = True
+        namspac2['bands_kpoints'].required = False
+
+        namspac3 = spec.inputs.create_port_namespace('second_relax_inputs')
+        namspac3.absorb(CommonRelaxInputGenerator.spec().inputs, exclude=('structure'))
+        namspac3['protocol'].non_db = True
+        namspac3['spin_type'].non_db = True
+        namspac3['relax_type'].non_db = True
+        namspac3['electronic_type'].non_db = True
+        namspac3['magnetization_per_site'].non_db = True
+        namspac3['threshold_forces'].non_db = True
+        namspac3['threshold_stress'].non_db = True
+        namspac3['engines']['relax']['options'].non_db = True
+        for key in namspac3:
+            namspac3[key].required = False
+            namspac3[key].populate_defaults = False
+            namspac3[key].default = ()
+        namspac3['relax_type'].required = True
+        namspac3['relax_type'].default = RelaxType.NONE
+
+        spec.input('relax_sub_process_class',
+                valid_type=orm.Str,
+                serializer=to_aiida_type,
+                validator=validate_sub_process_class_r
+                )
+        spec.input('bands_sub_process_class',
+                valid_type=orm.Str,
+                serializer=to_aiida_type,
+                validator=validate_sub_process_class_b
+                )
+
+        spec.inputs.validator = validate_inputs
+
+        spec.outline(
+            cls.initialize,
+            cls.run_relax,
+            cls.prepare_bands,
+            if_(cls.should_run_other_scf)(
+                cls.fix_inputs,
+                cls.run_relax
+            ),
+            cls.run_bands,
+            cls.inspect_bands
+        )
+
+        spec.output('final_structure', valid_type=orm.StructureData, help='The final structure.')
```

Comment:
Personally not a fan of these prefixes unless they are really necessary to distinguish them because there are multiple output structures.
```suggestion
        spec.output('structure', valid_type=orm.StructureData, help='The final structure.')
```